lib/lapis_lazuli/generators/cucumber/template/config/cucumber.yml

```diff
@@ -23,12 +22,41 @@ firefox: BROWSER=firefox
 chrome: BROWSER=chrome
 ie: BROWSER=ie
 safari: BROWSER=safari
+edge: BROWSER=edge
 
 
 ################################################################################
 # Listed environments (default is set in config.yml)
-t: TEST_ENV=test -t @t,@test
-test: TEST_ENV=test -t @t,@test
+t: -p test
+test: TEST_ENV=test -t @t,@test,@all_env,@all_environments -t ~@disabled_on_test
```

Comment:
Im quite puzzled and confused with all these tags, can we simplify this some more?

Reply:
I understand it might look confusing. But this allows for the most dynamic tag usage. With 1 tag you can either include the scenario in all environments, only test, or only exclude test from it. `@all_env,@all_environments` is so that you can use a short cut if you'd like. Same as `@t,@test`
It reminds me, Cucumber showed a message about this that they are deprecating it version 3.